### PR TITLE
W-11680259: avoid throwing remote context error in canParse method

### PR DESCRIPTION
**Pull request by arielmirra**
tests for Core PR https://github.com/aml-org/amf-core/pull/581

### CODE REVIEW

File: amf-cli/shared/src/test/scala/amf/validation/AMFModelAssertionTest.scala [hunk 1]
```diff
@@ -452,4 +452,9 @@ class AMFModelAssertionTest extends AsyncFunSuite with Matchers {
       getFirstRequestPayload(bu).annotations.lexical() shouldBe PositionRange(Position(13, 6), Position(15, 0))
     }
   }
+
```
Comment (tomsfernandez) [inline amf-cli/shared/src/test/scala/amf/validation/AMFModelAssertionTest.scala]: Couldn't this be a valid model test ?

Comment (arielmirra) [reply amf-cli/shared/src/test/scala/amf/validation/AMFModelAssertionTest.scala]: yes but I need to check that it conforms, and the modelAssertion method discards the results

**Review by nschejtman — DISMISSED** [review]

File: amf-cli/shared/src/test/scala/amf/validation/AMFModelAssertionTest.scala (hunk 1 above)
Comment (tomsfernandez) [reply amf-cli/shared/src/test/scala/amf/validation/AMFModelAssertionTest.scala]: hmm i didn't understand. The UniquePlatformValidationTest* or ValidModel* see if the parsing and validation report conform

Comment (arielmirra) [reply amf-cli/shared/src/test/scala/amf/validation/AMFModelAssertionTest.scala]: moved the test to UniquePlatformValidationTest

**Review by nschejtman — APPROVED** [review]

**Pull request merged**